FileSystem/FileIOTests.js

```diff
@@ -110,4 +161,5 @@ function file_open_close_tests() {
 function cleanup() {
   deleteFile('TESTFILE');
   deleteFile('PRINT_TEST');
+  deleteFile('WRITE_TEST');
```

Comment:
No tests for setFilePointer/GetFilePointer?

Reply:
Added tests